  Header header
  string loc_type
  string mailbox_color
  geometry_msgs/Point position